### PR TITLE
Added Node 14 support (#41)

### PR DESCRIPTION
**Pull request by EricRabil**
I only have a Mac, and am unsure how to build for Linux and Windows. However the Darwin build for Node 14 is working for me, I'll test it out on my projects and see how it fares.

Update: running fine for me on 14.1.0, however `npm run test` within the cWS repo is failing with the following:

```
ericrabil@Erics-Kitchen cWS % npm run test

> @clusterws/cws@1.6.0 test /Users/ericrabil/eric.git/cWS
> nyc mocha './tests/**/*.test.ts' --exit

  CWS Server & Client Tests SSL
dyld: lazy symbol binding failed: Symbol not found: __ZN4node10BaseObject12FromJSObjectEN2v85LocalINS1_5ValueEEE
  Referenced from: /Users/ericrabil/eric.git/cWS/dist/bindings/cws_darwin_83.node
  Expected in: flat namespace

dyld: Symbol not found: __ZN4node10BaseObject12FromJSObjectEN2v85LocalINS1_5ValueEEE
  Referenced from: /Users/ericrabil/eric.git/cWS/dist/bindings/cws_darwin_83.node
  Expected in: flat namespace
```

### CODE REVIEW

**Comment by EricRabil** [comment]
Just missing Windows bindings, adding 14 support was pretty easy I just need to compile them on the various platforms. I deployed cWS for Node 14 to my prod servers and its running fine.

**Comment by goriunov** [comment]
@EricRabil Thank you for your PR. Please rebase this branch on latest master also master includes fix for `Symbol not found` for Node 13 you may want to add the same for Node 14:

```
#if NODE_MAJOR_VERSION==13
  #include "headers/13/tls_wrap.h"
  #include "headers/13/base_object-inl.h" // Need add this line
#endif
```

File: make.bat [hunk 1]
```diff
@@ -12,11 +13,14 @@ if not exist targets (
...
 )
 
 cl /I targets/node-%v64%/include/node /I targets/node-%v64%/deps/uv/include /I targets/node-%v64%/deps/v8/include /I targets/node-%v64%/deps/openssl/openssl/include /I targets/node-%v64%/deps/zlib /EHsc /Ox /LD /Fedist/bindings/cws_win32_64.node src/*.cpp targets/node-%v64%/node.lib
 cl /I targets/node-%v72%/include/node /I targets/node-%v72%/deps/uv/include /I targets/node-%v72%/deps/v8/include /I targets/node-%v72%/deps/openssl/openssl/include /I targets/node-%v72%/deps/zlib /EHsc /Ox /LD /Fedist/bindings/cws_win32_72.node src/*.cpp targets/node-%v72%/node.lib
 cl /I targets/node-%v79%/include/node /I targets/node-%v79%/deps/uv/include /I targets/node-%v79%/deps/v8/include /I targets/node-%v79%/deps/openssl/openssl/include /I targets/node-%v79%/deps/zlib /EHsc /Ox /LD /Fedist/bindings/cws_win32_79.node src/*.cpp targets/node-%v79%/node.lib
+cl /I targets/node-%v83%/include/node /I targets/node-%v83%/deps/uv/include /I targets/node-%v83%/deps/v8/include /I targets/node-%v83%/deps/openssl/openssl/include /I targets/node-%v83%/deps/zlib /EHsc /Ox /LD /Fedist/bindings/cws_win32_79.node src/*.cpp targets/node-%v83%/node.lib
```
Comment (goriunov) [inline make.bat:23]: `/Fedist/bindings/cws_win32_79.node` should point to 83 -> `/Fedist/bindings/cws_win32_83.node`

**Comment by goriunov** [comment]
Also this branch should not include any bindings for other versions (except node 14)

**Comment by EricRabil** [comment]
Could you provide a bit more insight as to why you'd like bindings for versions earlier than 14 to be removed?

Node 12 is the latest LTS, we should at least include support for that.

**Comment by goriunov** [comment]
@EricRabil will you are rebuilding already existing bindings as they already exists in master branch there is no need to change anything in those. So after merging this branch it should just merge new bindings only for version 14. TBH i would prefer to rebuild all bindings including version 14 before releasing to npm to be on the safe side.

**Comment by EricRabil** [comment]
I removed the earlier bindings. I wasn’t able to build a windows binding for 14, but that’s because I couldn’t get a Windows development environment set up.

**Comment by goriunov** [comment]
@EricRabil better not to delete those bindings but remove changes which were done (you should be able to just copy already existing one from master branch) :)

**Comment by EricRabil** [comment]
Ahh sorry, I’ll fix that.

**Comment by EricRabil** [comment]
@goriunov hi there, are there any other changes you need from me? :)

**Comment by goriunov** [comment]
Looks good, i will try to merge this PR soon.

**Pull request merged**